.travis.yml

```diff
@@ -3,7 +3,7 @@ rvm: 2.2.4
 env:
   global:
     - OPENSTUDIO_VERSION: 2.5.2 #see comment in test.sh if you change this to >= 2.5.1
-    - OPENSTUDIO_VERSION_SHA: 6aed934ea8
+    - OPENSTUDIO_VERSION_SHA: a5af93e7ed
```

Comment:
@nllong I made a new 2.5.2 package and updated the SHA here.  What else would we need to do when re-making the AMIs tomorrow?

Reply:
This seems to only apply to travis testing so I assume the OS SHA is listed somewhere else as well?